impl/pcc/plugin.go

```diff
@@ -78,7 +78,7 @@ func (c *BasicPlugin) Run(cliConnection plugin.CliConnection, args []string) {
 // GetMetadata provides metadata about the CF plugin including a helptext for the user
 func (c *BasicPlugin) GetMetadata() plugin.PluginMetadata {
 	return plugin.PluginMetadata{
-		Name:    "pcc",
+		Name:    "gemfire",
```

Comment:
Note that this go file is in "impl/pcc/". Should it be changed to "impl/gemfire"?

Reply:
Yeah, it probably makes sense to rename the packages as well.

Reply:
Yes, this should be changed.